picker.py

```diff
@@ -324,4 +329,4 @@ def get_tags(keywords):
         picker.pick(submission)
         print(picker.get_results())
 
-import picker
\ No newline at end of file
+import picker
```

Comment:
This line seems odd to me. Does something requires it or can we delete it?

Reply:
Hm, I didn't add the line, but after testing it out, we should be able to delete it without changing the operation of the script.

Reply:
Consider it done :).